frontend/packages/client/src/hooks/useUserRoleOnCommunity.js

```diff
@@ -36,7 +36,7 @@ export default function useUserRoleOnCommunity({
     return false;
   }
 
-  if (loading || roles.length === 0) {
+  if (loading || roles.length === 0 || !communityId) {
```

Comment:
adding this check bc communityId could be undefined or null if called with params that are results of other hooks